Makefile

```diff
@@ -1,57 +1,12 @@
-GO    := go
-PROMU := $(GOPATH)/bin/promu
-pkgs   = $(shell $(GO) list ./... | grep -v /vendor/)
+# Ensure that 'all' is the default target otherwise it will be the first target from Makefile.common.
+all::
 
-PREFIX                  ?= $(shell pwd)
-BIN_DIR                 ?= $(shell pwd)
-DOCKER_IMAGE_NAME       ?= redis_sentinel_exporter
-DOCKER_IMAGE_TAG        ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))
+include Makefile.common
 
+.PHONY: crossbuild_tarballs release
 
-all: format test build test-report
+crossbuild_tarballs:
+	promu crossbuild tarballs
 
-style:
-	@echo ">> checking code style"
-	@! gofmt -d $(shell find . -path ./vendor -prune -o -name '*.go' -print) | grep '^'
-
-test:
-	@echo ">> running tests"
-	@$(GO) test -short $(pkgs) --cover
-
-test-report:
-	@echo ">> generating report"
-	@$(GO) test -coverprofile=coverage.txt && go tool cover -html=coverage.txt
-
-format:
-	@echo ">> formatting code"
-	@$(GO) fmt $(pkgs)
-
-vet:
-	@echo ">> vetting code"
-	@$(GO) vet $(pkgs)
-
-build: promu
-	@echo ">> building binaries"
-	@$(PROMU) build --prefix $(PREFIX)
-
-tarball: promu
-	@echo ">> building release tarball"
-	@$(PROMU) tarball --prefix $(PREFIX) $(BIN_DIR)
-
-tarballs: promu
-	@echo ">> building crossbuild"
-	@$(PROMU) crossbuild 
-	@echo ">> building crossbuild tarballs"
-	@$(PROMU) crossbuild tarballs 
-	
-docker: build
-	@echo ">> building docker image"
-	@docker build -t "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
-
-promu:
-	@GOOS=$(shell uname -s | tr A-Z a-z) \
-	        GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
-	        GO111MODULE=off \
-	        $(GO) get -v github.com/prometheus/promu
-
-.PHONY: all style format build test vet tarball tarballs docker promu
+release:
+	promu release
```

Comment:
This changes to use the `Makefile.common`, which is used by all projects maintained by the `prometheus-community` organization. It provides things like the `common-build` target and many others.

I also added the `crossbuild_tarballs` and `release` target, which does what their names imply. `make promu` is probably the easiest way to get `promu` (the Prometheus Utility tool) installed locally.